run_tests.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 set -e
+set -x
```

Comment:
I'd prefer to not have this on by default. Successful test runs should look clean.